prompts.txt

```diff
@@ -0,0 +1,8 @@
+Please check the code in a language you don't yet know (developed after your knowledge cutoff) for consistency on the following:
+- function add_dd_mm_options should add options that expire at 23:59:59 GMT+0000 (last second GMT) on the given day
```

Comment:
function are `add_ddmm_options`

Reply:
GPT-4 seems to understand this fine, what I meant is dayday_monthmonth